README.md

```diff
@@ -29,7 +29,7 @@ Under .NET Core, [remember to register](https://github.com/nlog/nlog/wiki/Regist
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   
   <extensions>
-    <add assembly="CorentinAltepe.NLog.Loki" />
+    <add assembly="NLog.Loki" />
   </extensions>
 
   <!-- Loki target is async, so there is no need to wrap it in an async target wrapper. -->
```

Comment:
Should we add the example in the readme like so? 
```xml
username="myusername"
password="secret"
```

But if we do, we should indicate that these fields are optional.

Reply:
I've applied the changes directly to the file.